packages/sui-test-e2e/package.json

```diff
@@ -17,7 +17,8 @@
     "@testing-library/cypress": "8.0.1",
     "commander": "8.2.0",
     "cypress": "8.6.0",
-    "cypress-file-upload": "5.0.8"
+    "cypress-file-upload": "5.0.8",
+    "har-validator": "5.1.5"
```

Comment:
😂

Reply:
It is not necessary
https://github.com/cypress-io/request/pull/15#issuecomment-979412776

Reply:
:D Es que no pasaba GitHub Actions sin la dependencia...